packages/run-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -227,164 +312,169 @@ export const makeVaultManager = (
       getAmountIn(quote),
     );
 
-    outstandingQuote = undefined;
+    state.outstandingQuote = null;
```

Comment:
It's debatable and I don't think we have coding standards on this. My reasoning was that `defineKind` requires all keys that will ever be on `state` to be on the object that `initState` returns. They could be `foo: undefined` but that is about the same as not being there because missing keys also resolve to `undefined`. So I think that since we do have to explicitly define them we're better off using `null` as a habit to convey that.

Another tact is to appeal to language: the outstanding quote _is_ defined, to be null.